ci/test/mkpipeline.py

```diff
@@ -139,6 +139,22 @@ def inputs(self) -> Set[str]:
         return inputs
 
 
+def prioritize_pipeline(pipeline: Any) -> None:
+    """Prioritize builds against main or release branches"""
+
+    branch = os.environ["BUILDKITE_BRANCH"]
+    priority = None
+
+    if branch.startswith("v"):
```

Comment:
Do we have release branches in form of branches? Or are they tags?

Reply:
They're tags, yep, but Git and Buildkite conflate tags and branches in a few places. This might be one of those places.

Reply:
Looking deeper I do think `BUILDKITE_TAG` would work better here.

Reply:
Unfortunately for the Nightly against a release tag the `BUILDKITE_TAG` is empty, only the `BUILDKITE_BRANCH` contains the version number. See https://buildkite.com/materialize/nightlies/builds/2841#0189562c-0bbd-4820-9298-c9095765ddcd

Reply:
We haven't had a nightly get triggered via the new trigger job yet. That trigger job should set `BUILDKITE_TAG` appropriately.

Reply:
`BUILDKITE_TAG` is also empty for CI builds against `main` , so I have taken a hybrid approach.